yarn-project/circuit-types/src/prover_coordination/epoch_proof_quote_payload.ts

```diff
@@ -50,9 +52,18 @@ export class EpochProofQuotePayload implements Signable {
     );
   }
 
+  static TYPE_HASH = Buffer32.fromBuffer(
+    keccak256(
+      Buffer.from(
+        'EpochProofQuote(uint256 epochToProve,uint256 validUntilSlot,uint256 bondAmount,address prover,uint32 basisPointFee)',
+      ),
+    ),
+  );
+
   getPayloadToSign(): Buffer {
```

Comment:
Should be updated similar to the EIP712.

Reply:
Actually I think this one is correct.

Reply:
Let me know how you think it should change.

Reply:
Actually, since we just consult the rollup for the digest, this can simply be removed.